.github/workflows/ci.yml

```diff
@@ -58,14 +58,15 @@ jobs:
       run: |
         yum -y install gcc make
         make REDIS_CFLAGS='-Werror'
-        
+
   build-freebsd:
```

Comment:
i think we can remove this one from the CI, and rely only on the daily.
it takes 5 minutes (most of the time to spin up the VM), i think it is not "cost effective" (global warming).